### ISSUE TITLE
Allow setting different idle_threshold for fullscreen windows

### ISSUE THREAD

**Issue by AdamantGarth** [dunst-project/dunst#844]
`idle_threshold` is a very useful feature which I currently do not use because it requires me to reach for my mouse/keyboard to close a notification when I'm watching movies/videos in fullscreen. It would be very convenient to be able to set something like `fullscreen_idle_threshold = 0` to prevent this.

### Installation info

- Version: `1.6.1`
- Install type: `package`
- Distro and version: `Arch Linux`

**Comment by fwSmit** [comment]
Yes that might be a useful setting. I would like to implement it a bit differently though. I would implement idle_timeout as a rule and add a matcher for fullscreen. This way you could also let you set notifications to only appear for 1 second when in fullscreen. 

If you didn't know about it setting fullscreen=pushback may help as a workaround. This way notifications don't get shown at all in fullscreen.

**Comment by AdamantGarth** [comment]
> I would implement idle_timeout as a rule and add a matcher for fullscreen. This way you could also let you set notifications to only appear for 1 second when in fullscreen.

That would be even better, thanks!

> If you didn't know about it setting fullscreen=pushback may help as a workaround. This way notifications don't get shown at all in fullscreen.

This is how a lot of other notification daemons work and I had to patch them, when I used them, to make them show notifications over fullscreen. For me it's better to have to reach for mouse to close a notification in fullscreen than miss it altogether :)

**Comment by fwSmit** [comment]
> This is how a lot of other notification daemons work and I had to patch them, when I used them, to make them show notifications over fullscreen. For me it's better to have to reach for mouse to close a notification in fullscreen than miss it altogether :)

Okay got it, you'll hear more when it's implemented.  Could you help with testing when it's implemented?

**Comment by AdamantGarth** [comment]
Sure thing! I already have a build environment ready because Arch ships dunst 1.5.0 in repos and I wanted to try progress bars from 1.6.0.
